### PR TITLE
Add FAQ to README.md

### PR DESCRIPTION
**Pull request by jaxdesmarais**
### Reason for changes

This is a starting point for some initial FAQs of questions that may come up in GitHub issues - David and I talked about adding these to the repo so we can address some common questions that may come up as folks find the SDK. This addresses both the current availability of the SDK as well as a holding place for future contribution/feedback pre GA release. If there are other things we think may come up that should also be here, happy for any suggestions/recommendations!

### Summary of changes

- Updated README with FAQ

### Note
Once this PR is approved/feedback addressed, I will add these same notes to the Android repo.

### Checklist

- ~[ ] Added a changelog entry~

- @jaxdesmarais

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
...
 Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payment methods in your iOS app.
 
+## FAQ
+### Availability
+The SDK is currently in the development process. This product will being developed fully open source - throughout the development process, we welcome any and all feedback. We recommend using the SDK in the sandbox environment until an official general availability release is out as aspects of the SDK _will likely_ change as we develop the SDK. This README will be updated with an official release date as it becomes available.
+
+### Contribution
+As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue/PR with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
+
+
```
Comment (sarahkoop) [inline README.md:11]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
 
 Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payment methods in your iOS app.
 
+## FAQ
+### Availability
+The SDK is currently in the development process. This product will being developed fully open source - throughout the development process, we welcome any and all feedback. We recommend using the SDK in the sandbox environment until an official general availability release is out as aspects of the SDK _will likely_ change as we develop the SDK. This README will be updated with an official release date as it becomes available.
+
+### Contribution
+As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue/PR with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
```
Comment (sarahkoop) [inline README.md]:
```suggestion
As the SDK is moved to general availability, we will be adding a contribution guide for developers that would like to contribute to the SDK. If you have suggestions for features that you would like to see in future iterations of the SDK, please feel free to open an issue, PR, or discussion with suggestions. If you want to open a PR but are unsure about our testing strategy, we are more than happy to work with you to add tests to any PRs before work is merged.
```

Do we want people to use the GH discussions feature? It's kind of nice for sharing ideas

**Review by sarahkoop — APPROVED** [review]
Looks good to me - can we copy the same over to Android?

File: README.md (hunk 2 above)
Comment (jaxdesmarais) [reply README.md]: Yeah - we should for sure use the discussions feature for this!

**Comment by jaxdesmarais** [comment]
> Looks good to me - can we copy the same over to Android?

Yep, once this is approved, I plan to copy the content over to the Android repo 😄

File: README.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
 
 Welcome to PayPal's iOS SDK. This library will help you accept card, PayPal, Venmo, and alternative payment methods in your iOS app.
 
+## FAQ
+### Availability
+The SDK is currently in the development process. This product will being developed fully open source - throughout the development process, we welcome any and all feedback. We recommend using the SDK in the sandbox environment until an official general availability release is out as aspects of the SDK _will likely_ change as we develop the SDK. This README will be updated with an official release date as it becomes available.
```
Comment (sshropshire) [inline README.md]:
```suggestion
The SDK is currently in the development process. This product is being developed fully open source - throughout the development process, we welcome any and all feedback. Aspects of the SDK _will likely_ change as we develop the SDK. We recommend using the SDK in the sandbox environment until an official release is available. This README will be updated with an official release date once it is generally available.
```

**Review by sshropshire — APPROVED** [review]
Random nit. Overall LGTM&nbsp;🌊

**Pull request merged**